Lib/test/test_sysconfig.py

```diff
@@ -263,10 +278,14 @@ def test_get_config_h_filename(self):
         self.assertTrue(os.path.isfile(config_h), config_h)
 
     def test_get_scheme_names(self):
-        wanted = ['nt', 'posix_home', 'posix_prefix']
+        sys.path.append(os.path.abspath(os.path.join(__file__, '..', 'vendor_config')))
+        # force re-load of vendor schemes with the patched sys.path
+        sysconfig._load_vendor_schemes()
+
+        wanted = ['nt', 'posix_home', 'posix_prefix', 'some_vendor']
```

Comment:
Would it make sense to have two tests, one which patches sys.path and another that doesn't?

Reply:
Ditto.